src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -640,6 +649,20 @@ def make_acc_group(acc, make_random):
     yield from _make_group("account group", acc.config, acc.groups, make_random)
 
 
+@pytest.fixture
+def make_migrated_group(acc, ws, make_group, make_acc_group):
+    """Create a pair of groups in workspace and account. Assign account group to workspace."""
+
+    def inner():
+        ws_group = make_group()
+        acc_group = make_acc_group()
+        acc.workspace_assignment.update(ws.get_workspace_id(), acc_group.id, [WorkspacePermission.USER])
+        # need to return both, as acc_group.id is not in MigratedGroup dataclass
+        return MigratedGroup.partial_info(ws_group, acc_group), acc_group
+
+    return inner
```

Comment:
```suggestion
@pytest.fixture
def migrated_group(acc, ws, make_group, make_acc_group) -> MigratedGroup:
    ws_group = make_group()
    acc_group = make_acc_group()
    acc.workspace_assignment.update(ws.get_workspace_id(), acc_group.id, [WorkspacePermission.USER])
    return MigratedGroup.partial_info(ws_group, acc_group)
```
and place this into `tests/integration/workspace_access/conftest.py`. and this removes the need to do `migrated_group, _ = make_migrated_group()` everywhere.

This file should not have non-generic fixtures - we'll refactor them out to `pytester` library soon.

you need two return results only for test_scim, so copy-paste them there. otherwse `, _` everywhere is unreadable